app/point.py

```diff
@@ -17,7 +17,8 @@ def __hash__(self) -> int:
         # which is NOT a best practice, but you will be able to predict
         # a hash value by coordinates of the point and its index
         # in the hashtable as well
-        return hash((self.x, self.y))
+        # return hash((self.x, self.y))
+        return self.x + self.y
```

Comment:
The current hash function implementation `return self.x + self.y` is not suitable for production as it can lead to hash collisions. Consider using a tuple-based hash like `return hash((self.x, self.y))` for a more robust solution.